File: app/src/main/java/com/baktiyar/android/jardamberem/ui/product/feed_products/adapter/ViewPagerAdapter.java
package com.baktiyar.android.jardamberem.ui.product.feed_products.adapter;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentStatePagerAdapter;

import java.util.ArrayList;
import java.util.List;

public class ViewPagerAdapter extends FragmentStatePagerAdapter {

    private List<Fragment> mFragmentList = new ArrayList<>();
    private List<String> titles = new ArrayList<>();

    public ViewPagerAdapter(FragmentManager manager) {
        super(manager);
    }

    @Override
    public Fragment getItem(int position) {
        return mFragmentList.get(position);
    }

    @Override
    public int getCount() {
        return mFragmentList.size();
    }

    public void addFragment(Fragment fragment, String title) {
        mFragmentList.add(fragment);
        titles.add(title);
        notifyDataSetChanged();
    }

    @Override
    public CharSequence getPageTitle(int position) {
        return titles.get(position);
    }
}

File: app/src/main/java/com/baktiyar/android/jardamberem/ui/main/fragment/ViewPagerAdapter.kt
package com.baktiyar.android.jardamberem.ui.main.fragment

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentStatePagerAdapter
import com.baktiyar.android.jardamberem.R
import com.baktiyar.android.jardamberem.StartApplication


class ViewPagerAdapter(fm: FragmentManager, var fragments: Array<Fragment>) : FragmentStatePagerAdapter(fm) {
    override fun getCount(): Int = 2

    override fun getItem(position: Int): Fragment = fragments[position]

    override fun getPageTitle(position: Int): CharSequence? {
        return if (position == 0)
            StartApplication.INSTANCE.getString(R.string.give_help)
        else
            StartApplication.INSTANCE.getString(R.string.need_help)
    }
}